features/wp-config.feature

```diff
@@ -0,0 +1,96 @@
+Feature: wp-config
+  Scenario: Default WordPress install with wp-config.php
```

Comment:
I think this scenario is unneeded and only slows down testing without actually providing a benefit.